tests/unit/core/test_withExperiment.js

```diff
@@ -0,0 +1,111 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+
+import { withExperiment } from 'core/withExperiment';
+import { fakeCookie } from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  class SomeComponentBase extends React.Component {
+    render() {
+      return <div className="component" />;
+    }
+  }
+
+  function setComponentWithExperiment({ experimentProps } = {}) {
+    return withExperiment({
+      id: 'ABtest',
+      variantA: 'AName',
+      variantB: 'BName',
+      ...experimentProps,
+    })(SomeComponentBase);
+  }
```

Comment:
Can you merge this function within `renderComponentWithExperiment`?

Reply:
initially i had it separate but I need it separate for the displayName case

Reply:
oops sorry I wanted to remove this comment, it does not imply for the reason you gave

Reply:
ok cool, we're on the same page then : )